ceagle/wsgi.py

```diff
@@ -63,12 +69,3 @@ def index():
 @app.errorhandler(404)
 def not_found(error):
     return flask.jsonify({"error": "Not found"}), 404
-
-
-def main():
```

Comment:
keep this code, it's usefull if you would like to run app in dev mode: "python wsgi.py"